lib/plugin-webpack5.js

```diff
@@ -0,0 +1,200 @@
+      vueLoaderOptions.compilerOptions = vueLoaderOptions.compilerOptions || {}
+      vueLoaderOptions.compilerOptions.modules = vueLoaderOptions.compilerOptions.modules || []
+      vueLoaderOptions.compilerOptions.modules.push(progressiveLoaderModule)
+
+      const imageRuleIndex = this.rules.findIndex(rule => {
+        return rule.resource &&
+          !rule.resourceQuery &&
+          ['.png', '.jpg', '.jpeg', '.gif'].some(ext => rule.resource(ext))
+      })
+      let imageRule = this.rules[imageRuleIndex]
+
+      const options = typeof this.options.progressiveImages === 'boolean'
+        ? undefined
+        : this.options.progressiveImages
+
+      if (!imageRule) {
+        imageRule = {
+          test: /\.(png|jpe?g|gif|svg|eot|ttf|woff|woff2)(\?.*)?$/,
+          oneOf: [
+            {
+              test: /\.(png|jpe?g|gif)$/,
+              resourceQuery: options ? options.resourceQuery : /vuetify-preload/,
+              use: [
+                {
+                  loader: 'vuetify-loader/progressive-loader',
+                  options
+                },
+                {
+                  loader: 'url-loader',
+                  options: { limit: 8000 }
+                }
+              ]
+            },
+            {
+              loader: 'url-loader',
+              options: { limit: 8000 }
+            }
+          ]
+        }
+        rules.push(imageRule)
+      } else {
+        if (Array.isArray(imageRule.use)) {
+          imageRule.oneOf = [
+            {
+              test: /\.(png|jpe?g|gif)$/,
+              resourceQuery: options ? options.resourceQuery : /vuetify-preload/,
+              use: [
+                {
+                  loader: 'vuetify-loader/progressive-loader',
+                  options
+                },
+                ...imageRule.use
+              ]
+            },
+            ...imageRule.use
+          ]
+        } else if (imageRule.loader) {
+          imageRule.oneOf = [
+            {
+              test: /\.(png|jpe?g|gif)$/,
+              resourceQuery: options ? options.resourceQuery : /vuetify-preload/,
+              use: [
+                {
+                  loader: 'vuetify-loader/progressive-loader',
+                  options
+                },
+                {
+                  loader: imageRule.loader,
+                  options: imageRule.options
+                }
+              ]
+            },
+            {
+              loader: imageRule.loader,
+              options: imageRule.options
+            }
+          ]
+        }
+        delete imageRule.use
+        delete imageRule.loader
+        delete imageRule.options
+      }
+    }
+
+    rule.oneOf = [
+      {
+        resourceQuery: '?',
+        use: rule.use
+      },
+      {
+        use: [
+          {
+            loader: require.resolve('./loader'),
+            options: {
+              match: this.options.match || [],
+              attrsMatch: this.options.attrsMatch || []
```

Comment:
```suggestion
              attrsMatch: this.options.attrsMatch || [],
              registerStylesSSR: this.options.registerStylesSSR || false
```

Is `registerStylesSSR` deliberately not included in the webpack5 version or was this an oversight?

Reply:
oversight